buildSVG.js

```diff
@@ -33,9 +33,10 @@ fluid.removeChildren = function (node) {
 fluid.nodeToText = function (node, indentLevel) {
     var text = node.outerHTML;
     var formatted = jsBeautify.html(text, {
-        indent_level: indentLevel || 0
+        indent_level: indentLevel || 0,
+        end_with_newline: true
     });
-    return formatted;
+    return formatted.replace(/^\s*[\r\n]/gm, ""); // remove empty lines
```

Comment:
This regex will probably not work on Windows. Nosing around StackOverflow I found the following answer which seems plausible and well-approved - https://stackoverflow.com/a/39282395 - could you test it out and see if it is effective at least on your platform?

Reply:
The regex used was from another [StackOverflow article](https://stackoverflow.com/questions/16369642/javascript-how-to-use-a-regular-expression-to-remove-blank-lines-from-a-string) -- see "Edit 3" -- which claimed to work on all platforms.  It had the added benefit of not removing the newline at the end of the file, which would cause the lint error this is supposed to fix.

I can't test Windows.  I gather you have and discovered that the regex doesn't work?  I'll try out the other pattern with macOS.

Reply:
> I found the following answer which seems plausible and well-approved - https://stackoverflow.com/a/39282395 

That regex isn't right because it removes the last line causing lint to report "Expected a newline at the end of the file"

I've looked around and tried a couple of other regular expressions, and eventually found a page that used a function that should work everywhere -- a sequence of `split("\n"),` `trim()` each array element, discard empty elements, and finally` join("\n")` what's left.

However, `js-beautify` has a configuration property `preserve_newlines` when set that to `false` does what's needed.  The latest code uses that property and forgoes any regex.